google/cloud/forseti/services/server_config/service.py

```diff
@@ -26,6 +28,64 @@
 LOGGER = logger.get_logger(__name__)
 
 
+def _run(client):
+    """Runs Forseti.
+
+    Args:
+        client (ClientComposition): Forseti client object.
+
+    Yields:
+        ServerRunReply: the returned proto message.
+    """
+
+    model_name = 'run_model'
```

Comment:
I think the model name should be dynamic. Maybe prefix it with RunModel and append the timestamp to the end?

Reply:
Yes.  That is a very good suggestion. Will do.